### ISSUE TITLE
Add documentation for "bundling" python module(s)

### ISSUE THREAD

**Issue by jcfr** [KitwareMedical/SlicerCustomAppTemplate#54]
Related pull requests and issues:
* https://github.com/Slicer/Slicer/pull/6913
* https://github.com/Slicer/Slicer/issues/7171

**Comment by benzwick** [comment]
Is there a way to include/bundle python packages in a Slicer Custom App so that they don't need to be downloaded at run time (which is the suggested solution in the links above)?

**Comment by ebrahimebrahim** [comment]
Yes, in a custom app it can be done! I don't think this is necessarily the solutions linked above but here is an example where I recently did it:

- added this python dependencies project to the top level cmake: https://github.com/OpenwaterHealth/OpenLIFU-app/blob/main/CMakeLists.txt#L175
- added this cmake file to define the project: https://github.com/OpenwaterHealth/OpenLIFU-app/blob/main/SuperBuild%2FExternal_python-sliceropenlifu-requirements.cmake 
  - Here you can see the pip install command referencing a requirements.txt file that is contained inside a slicer extension, but you could for example make it point to a file that is in your custom app repo
